src/Text/Markdown/SlamDown/Pretty.purs

```diff
@@ -128,17 +128,20 @@ prettyPrintDate { day, month, year } =
     <> "-"
     <> printIntPadded 2 day
 
-prettyPrintTime ∷ SD.TimeValue → String
-prettyPrintTime { hours, minutes }=
+prettyPrintTime ∷ SD.TimePrecision → SD.TimeValue → String
+prettyPrintTime prec { hours, minutes, seconds }=
```

Comment:
Have you tried it with halogen in chrome? I found it makes an error w/o `T` separator.